test/e2e/framework/statefulset/fixtures.go

```diff
@@ -0,0 +1,278 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package statefulset
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"sort"
+	"strconv"
+
+	apps "k8s.io/api/apps/v1"
+	"k8s.io/api/core/v1"
```

Comment:
nit [1]: i think including the version to import aliases is useful for reviewing code. i.e., 
```
appsv1 "k8s.io/api/apps/v1"
corev1 "k8s.io/api/core/v1"
```
instead of 
```
apps "k8s.io/api/apps/v1"
	"k8s.io/api/core/v1"
```